contents/handbook/engineering/developing-locally.md

```diff
@@ -45,20 +45,25 @@ Windows isn't supported natively. But, Windows users can run a Linux virtual mac
 
 In case some steps here have fallen out of date, please tell us about it – feel free to [submit a patch](https://github.com/PostHog/posthog.com/blob/master/contents/handbook/engineering/developing-locally.md)!
 
-## Developing with Codespaces
-
-This is a faster alternative to get up and running. If you don't want to or can't use Codespaces continue from the next section.
+## Option 1: Developing with Codespaces
 
+This is a faster option to get up and running. If you don't want to or can't use Codespaces, continue from the next section.
 
 1. Create your codespace.
 ![](https://user-images.githubusercontent.com/890921/231489405-cb2010b4-d9e3-4837-bfdf-b2d4ef5c5d0b.png)
-2. Update it to 8-core machine type (the smallest is probably too small to get PostHog running properly). Consider also changing "Open in ..." to be your favorite editor.
+2. Update it to 8-core machine type (the smallest is probably too small to get PostHog running properly).
 ![](https://user-images.githubusercontent.com/890921/231490278-140f814e-e77b-46d5-9a4f-31c1b1d6956a.png)
-3. Open a terminal window and run `docker compose -f docker-compose.dev.yml up`.
-4. Open a terminal window and run `./bin/migrate` and then `./bin/start`.
-5. Open browser to http://localhost:8000/.
+3. Open the codespace, using one of the "Open in" options from the list.
+4. In the codespace, open a terminal window and run `docker compose -f docker-compose.dev.yml up`.
+5. Also in the codespace, open another terminal window and run `./bin/migrate` and then `./bin/start`.
+6. Open browser to http://localhost:8000/.
+7. To get some practical test data into your brand-new instance of PostHog, run `DEBUG=1 ./manage.py generate_demo_data`.
+
+## Option 2: Developing locally
 
-## macOS prerequisites
+## Prerequisites
```

Comment:
I'd prefer if these two weren't the same heading level, Prerequisites should probably be h3 here